src/engine/cuecontrol.cpp

```diff
@@ -922,6 +926,8 @@ void CueControl::updateIndicators() {
                         if (cueMode == CUE_MODE_MIXXX) {
                             // in Mixxx mode Cue Button is flashing slow if CUE will move Cue point
                             m_pCueIndicator->setBlinkValue(ControlIndicator::RATIO1TO1_500MS);
+                        } else if (cueMode == CUE_MODE_MIXXX_NO_BLINK) {
```

Comment:
Whoops, I meant to have m_pCueIndicator->setBlinkValue(ControlIndicator::OFF); here. I'm not sure what happened to it, but I put it in the next commit.